### DISCUSSION TITLE
How to avoid code duplication?

### DISCUSSION THREAD

**Discussion by i3e-ai** [getbem/getbem.github.io#258] [Q&A]
I have two set of same representative code due to block and I feel it's duplicate. How can avoid duplicate CSS code as well having different block elements? An example would be very much helpful.

**Comment by tadatuta** [comment]
You may use mixes:

```html
<div class="my-block">
    <div class="my-block__elem1 my-block__common"></div>
    <div class="my-block__elem2 my-block__common"></div>
</div>
```

For more info about mixes see https://en.bem.info/methodology/css/#mixes
